source/docs/api-reference/wpilib-integration/gyro-integration.rst

```diff
@@ -0,0 +1,20 @@
+Gyro Integration
+================
+
+CTR-Electronics supported gyroscopes (Pigeon 2) implement the WPILib ``Gyro`` (`Java <https://github.wpilib.org/allwpilib/docs/release/java/edu/wpi/first/wpilibj/interfaces/Gyro.html>`__, `C++ <https://github.wpilib.org/allwpilib/docs/release/cpp/classfrc_1_1_gyro.html>`__) interface. Phoenix Pro implements all inherited functions:
+
+- ``reset``
+- ``getAngle``
+- ``getRate``
+- ``getRotation2d``
```

Comment:
Same as in motor controller integration, I don't think we should list off every function in the Gyro API, as we shouldn't have to update our docs when functions are added to or removed from the WPILib API. We already link to the API docs, and since it's an interface, we have to have implementations for all of those functions.